### ISSUE TITLE
Collective Thought Network effect listed as "Unknown"

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#2246]
### Environment
* **FreeOrion Version**: 0.4.8 RC3 2018-07-31
* **Operating System**: Windows 7
* **Fetched as**
  * Binary release

### Description
The effect of Collective Thought Network is missing a description. Its effect (i.e. +0.5*pop, unmodified by species trait, to research or industry if corresponding focus is set) is listed as "Unknown".

Examples from the attached save game:

Chato homeworld (Miyamoto) with research focus:
```
Research:
+0.20*pop : Research Focus
+0.50*pop : Pure-Energy Metabolism
+0.10*pop : Alg. Elegance
+0.10*pop : Dist. Thought Computing
+0.75*pop : Enclave of the Void
+1.65*pop : Great research (i.e. +100% of current subtotal)
+0.50*pop : Quantum Networking
+0.20*pop : Stellar Tomography
+2        : Nascent Artificial Intelligence
+5        : Automated History Analyser
+5        : Cultural Archives
+0.50*pop : UNKNOWN
Industry:
+5        : Adaptive Automation
+0.50*pop : Cultural Archives
```

Human homeworld (Vargas Beta I) with industry focus, asteroid belt and no gas giant:
```
Research:
+0.10*pop : Dist. Thought Computing
+2        : Nascent Artificial Intelligence
+5        : Automated History Analyser
+5        : Cultural Archives
Industry:
+0.20*pop : Industry Focus
+0.20*pop : Pure-Energy Metabolism
+0.20*pop : Fusion Generation
+0.60*pop : Industrial Center III
+0.10*pop : Robotic Production
+5        : Microgravity Industry
+5        : Adaptive Automation
+0.50*pop : Cultural Archives
+0.40*pop : Solar Orbital Generator
+0.50*pop : UNKNOWN
```

### Steps to reproduce
Load attached save game and check any planet with research or industry focus.
[FreeOrion_Sicofonte_Foundation_0240_20180812_141525.zip](https://github.com/freeorion/freeorion/files/2280955/FreeOrion_Sicofonte_Foundation_0240_20180812_141525.zip)

**Comment by MatGB** [comment]
It's an old intermittent issue and doesn't normally happen with a freshly
loaded save. It's been about for as long as I can remember but not easily
reproducible.

I think it's a client server info miss match issue, the client thinks
there's movement within radius stopping the effect, the server doesn't. So
you get an "unknown" accounting label because the client can't explain why
the server insists on a different number.

I've never seen it as a negative unknown though, and if I'm right that
should sometimes happen.

**Comment by Oberlus** [comment]
@Vezzra 
> did you upgrade to another build while playing that game?
I didn't.

> ran a couple of quick tests, and I can't reproduce the issue
May I assume you used some trick (like supertester or fast research/production) to get fast to the CTN? Maybe supertesters' OP detection is "solving" that possible missmatch pointed out by @MatGB.

However, I can't understand the issue. In that save game I have several spare ships on the empty systems around the CTN building and have plenty of detection strength, so no krill or whatever is wandering around (at least not for the 3+ consecutive turns which I checked that consistently produced the Unknown effect). If the Client is detecting a moving ship within 200 radius around the CTN building, it is not showing it to me and that would be weird and serious bug (client sees something that does not show to the player, and server does not see it anyway).

A blind shot: could it be that the client (and not the server) is considering a stationary ship as moving (like moving from current system to current system)?

**Comment by geoffthemedio** [comment]
> A blind shot: could it be that the client (and not the server) is considering a stationary ship as moving (like moving from current system to current system)?

Or some lingering turn timing issue. You could try removing the stationary requirement from the EffectsGroup activation condition.

**Comment by Vezzra** [comment]
> May I assume you used some trick (like supertester or fast research/production) to get fast to the CTN?

Yep, I did. Super Tester, fast research/production, the whole thing.

> Maybe supertesters' OP detection is "solving" that possible missmatch pointed out by @MatGB.

That is a very likely possibility.

Whatever is going on here, is certainly a weird bug. But apparently not something we'll be able to fix quickly...
